taskcluster/translations_taskgraph/transforms/cached_tasks.py

```diff
@@ -6,12 +6,6 @@
 # It exists because there are two features that we need that are missing upstream:
```

Comment:
One feature left :)

Reply:
Indeed! I intend to follow up with this at some point.